### ISSUE TITLE
Show OpenDNS categorisation within results

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/cmp-issues#13]
Showing the OpenDNS categorisation may help indicate why a result has been blocked at a provider.

**Comment by JimKillock** [comment]
Should we dedicate a probe to OpenDNS?

**Comment by dantheta** [comment]
I was thinking that we could categorise urls on the server at submission time using opendns. It would be pretty easy to run orgprobe against opendns too for actual blocking results, although we can infer that from the category.

**Comment by graphiclunarkid** [comment]
There's conflicting information on the OpenDNS website about whether we can do this.

They say non-commercial re-use is [coming soon](http://community.opendns.com/domaintagging/faq/#MayIUseTheDomainTaggingDataElsewhere) and they also say scraping the site is [not allowed](https://support.opendns.com/entries/24423704-Domain-Tagging-Toolbar-for-Firefox) (last entry on the page says "Please do not make scripts that abuse our API as such is a bannable offense as stated in our terms of service.")

We could contact them to ask if they'd let us talk to their API from ours if people thought it would be worthwhile.

**Comment by JimKillock** [comment]
I agree contacting them would be a good idea, rather than just doing it

**Comment by graphiclunarkid** [comment]
Now that we have DMOZ integration does anyone think this is worth pursuing or has it been superseded?

**Comment by JimKillock** [comment]
This is hard to say for me. We have a _lot_ of data that is inoffensive but may not be easily parsed especially from outside. Maybe you could find out how much of the data we have is not matched by a DMOZ categorisation? If it is low then it is probably not worth pursuing.

**Comment by dantheta** [comment]
Currently, 1211818 of 3479133 URLs (34%) do not have a DMOZ category, (including some urls which came from DMOZ, which is a little worrying - I'll look into that).
